pokemon.go

```diff
@@ -67,7 +67,7 @@ func (p *Pokemon) HasValidEVs() bool {
 // implement Stringer
 
 // display a Pokemon close to how it would appear in a Pokemon battle
-func (p *Pokemon) String() string {
+func (p Pokemon) String() string {
 	return fmt.Sprintf("%v%v\tLv%d\nHP: %d/%d\n", p.GetName(),
 		p.Gender, p.Level, p.CurrentHP, p.Stats[0])
```

Comment:
```suggestion
		p.Gender, p.Level, p.CurrentHP, p.Stats[STAT_HP])
```

while we're editing this might as well right?

Reply:
agree